web-analogs.md

```diff
@@ -0,0 +1,1035 @@
+---
+layout: code_comp_page
+title: HTML/CSS Analogs in Flutter
+sidebar: home_sidebar
+permalink: /web-analogs/
+---
+
+* TOC Placeholder
+{:toc}
+
+<div class="begin-examples"></div>
+This page is for users who are familiar with the HTML and CSS syntax for 
+arranging components of an application's UI. It maps HTML/CSS code snippets (on 
+the left) to their Flutter/Dart code equivalents (on the right).
+
+## Performing Basic Layout Operations
+
+The following examples show how to perform the most common UI layout tasks.
+
+### Aligning Text
+
+In both HTML and Flutter, by default child elements or widgets are anchored at 
+the top left.
+<div class="lefthighlight">
+{% prettify css %}
+<div class="greybox">
+    Lorem ipsum
+</div>
+
+.greybox {
+      background-color: #e0e0e0; /* grey 300 */
+      width: 320px;
+      height: 240px;
+      font: 900 24px Roboto;
+    }
+{% endprettify %}
+</div>
+<div class="righthighlight">
+{% prettify dart %}
+  var container = new Container( // grey box
+    child: new Text(
+      "Lorem ipsum",
+      style: bold24Roboto,
+    ),
+    width: 320.0,
+    height: 240.0,
+    decoration: new BoxDecoration(
+      backgroundColor: Colors.grey[300],
+    ),
+  );
+{% endprettify %}
+</div>
+
+### Setting Background Color
+In Flutter, you set background color in a [Container](https://docs.flutter.io/flutter/widgets/Container-class.html)’s
+“decoration” property.
```

Comment:
Yes, good one. I addressed my reluctance by using code font for only the Flutter properties, not the CSS ones.